app/models/viewModels/LateSubmissionPenaltyViewModel.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.viewModels
+
+import play.api.i18n.Messages
+import play.api.libs.json.{Json, OFormat}
+import views.templates.payments.PaymentMessageHelper
+
+import java.time.LocalDate
+
+case class LateSubmissionPenaltyViewModel(chargeType: String,
```

Comment:
Can you add a test spec for this view model please. Looks as though a good range of tests could target: the `description` function, the `makePaymentRedirect` val, and the JSON reads/writes.

Could you also add one test case to the existing set of tests in `ChargeDetailsViewModelSpec` that test the `title` function. The function is well tested already but seeing as we have one for every other type of view model, might as well keep that exhaustive.